tests/tests/alchemy_api.rs

```diff
@@ -22,19 +22,19 @@ async fn test_token_balances(#[future] erc20: (Katana, KakarotEvmContract), _set
     let eoa = katana.eoa();
     let eoa_address = eoa.evm_address().expect("Failed to get Eoa EVM address");
     let erc20_address: Address =
-        Into::<Felt252Wrapper>::into(erc20.evm_address).try_into().expect("Failed to convert EVM address");
+        Felt252Wrapper::from(erc20.evm_address).try_into().expect("Failed to convert EVM address");
 
     let (server_addr, server_handle) =
         start_kakarot_rpc_server(&katana).await.expect("Error setting up Kakarot RPC server");
 
     // When
-    let to = EthersAddress::from_slice(eoa.evm_address().unwrap().as_slice());
+    let to = ethers::abi::Address::from_slice(eoa.evm_address().unwrap().as_slice());
```

Comment:
should we create an issue to delete ethers as a dependency and only use alloy?

Reply:
I think @tcoratger tried but iirc the abigen tooling for alloy isn't quite there yet

Reply:
Yes we should, that will be the first step: https://github.com/kkrt-labs/kakarot-rpc/pull/958

Then will focus on the abigen stuff to remove that and replace by alloy.